activities.serge.json

```
@@ -274,5 +274,28 @@
       "sv-se sv",
       "tr-tr tr"
     ]
+  },
+  {
+    "name": "activityEvaluationEditor",
+    "parser_plugin": {
+      "plugin": "parse_js"
+    },
+    "source_match": "en\\.js$",
+    "source_dir": "components/d2l-activity-evaluation-editor/lang",
```

Does this need to have the `d2l-activity-editor` as well?

```suggestion
    "source_dir": "components/d2l-activity-editor/d2l-activity-evaluation-editor/lang",
```

---

Good catch. Thank you!